From 9be4157391edf0c5fc4ee36adfb1eb1c3bdb8e3b Mon Sep 17 00:00:00 2001
From: Yaakov Selkowitz <yselkowitz@users.sourceforge.net>
Date: Wed, 17 Mar 2010 13:33:39 -0500
Subject: [PATCH 14/37] Respect value of SED from configure

We now use libtool, which calls AC_PROG_SED and sets SED as the path to
a fully-functional 'sed' (which may also be called 'gsed' if GNU sed is
installed alongside a proprietary version).  Therefore we should respect
the value of SED so we are sure to use the correct one.

Signed-off-by: Yaakov Selkowitz <yselkowitz@users.sourceforge.net>
Reviewed-by: Alan Coopersmith <alan.coopersmith@sun.com>
Reviewed-by: Julien Cristau <jcristau@debian.org>
---
 cpprules.in               |    2 --
 hw/dmx/Makefile.am        |    2 +-
 hw/dmx/config/Makefile.am |    2 +-
 3 files changed, 2 insertions(+), 4 deletions(-)

diff --git xorg-server-1.7.99.902/cpprules.in xorg-server-1.7.99.902/cpprules.in
index 2e0856b..301305e 100644
--- xorg-server-1.7.99.902/cpprules.in
+++ xorg-server-1.7.99.902/cpprules.in
@@ -2,8 +2,6 @@
 # Rules for generating files using the C pre-processor
 # (Replaces CppFileTarget from Imake)
 
-SED = sed
-
 SUFFIXES = .pre .man .man.pre
 
 # Translate XCOMM into pound sign with sed, rather than passing -DXCOMM=XCOMM
diff --git xorg-server-1.7.99.902/hw/dmx/Makefile.am xorg-server-1.7.99.902/hw/dmx/Makefile.am
index cdee586..fc9fde7 100644
--- xorg-server-1.7.99.902/hw/dmx/Makefile.am
+++ xorg-server-1.7.99.902/hw/dmx/Makefile.am
@@ -109,7 +109,7 @@ MAN_SUBSTS = \
 SUFFIXES = .$(APP_MAN_SUFFIX) .man
 
 .man.$(APP_MAN_SUFFIX):
-	sed $(MAN_SUBSTS) < $< > $@
+	$(SED) $(MAN_SUBSTS) < $< > $@
 
 EXTRA_DIST = $(appman_PRE)
 
diff --git xorg-server-1.7.99.902/hw/dmx/config/Makefile.am xorg-server-1.7.99.902/hw/dmx/config/Makefile.am
index 9fa7113..7de745f 100644
--- xorg-server-1.7.99.902/hw/dmx/config/Makefile.am
+++ xorg-server-1.7.99.902/hw/dmx/config/Makefile.am
@@ -69,7 +69,7 @@ XORGRELSTRING = @PACKAGE_STRING@
 MAN_SUBSTS = -e 's|__vendorversion__|"$(XORGRELSTRING)" "$(XORGMANNAME)"|' 
 
 .man.$(APP_MAN_SUFFIX):
-	sed $(MAN_SUBSTS) < $< > $@
+	$(SED) $(MAN_SUBSTS) < $< > $@
 
 EXTRA_DIST = \
 	$(appman_PRE) \
-- 
1.7.0.2

